imagestreams/mariadb-centos.json

```diff
@@ -44,6 +44,42 @@
           "type": "Local"
         }
       },
+      {
+        "name": "10.11-c9s",
+        "annotations": {
+          "openshift.io/display-name": "MariaDB 10.11 (CentOS 9 Stream)",
+          "openshift.io/provider-display-name": "Red Hat, Inc.",
+          "description": "Provides a MariaDB 10.11 database on CentOS 9 Stream. For more information about using this database image, including OpenShift considerations, see https://github.com/sclorg/mariadb-container/tree/master/10.11/README.md.",
+          "iconClass": "icon-mariadb",
+          "tags": "database,mariadb",
+          "version": "10.11"
+        },
+        "from": {
+          "kind": "DockerImage",
+          "name": "quay.io/sclorg/mariadb-1011-c9s:latest"
+        },
+        "referencePolicy": {
+          "type": "Local"
+        }
+      },
+      {
+        "name": "10.11-c8s",
```

Comment:
name should be "10.11-el8"

Reply:
As you can see here: https://quay.io/organization/sclorg all our container that are part of CentOS Stream has the name either 'c9s' or 'c8s'
And we do not want to rename all of them.